stripe/_search_result_object.py

```diff
@@ -121,3 +150,22 @@ def next_search_result_page(
         return self._search(
             **params_with_filters,
         )
+
+    async def next_search_result_page_async(
+        self, **params: Unpack[RequestOptions]
+    ) -> Self:
+        if not self.has_more:
+            options, _ = extract_options_from_dict(params)
+            return self._empty_search_result(
+                api_key=options.get("api_key"),
+                stripe_version=options.get("stripe_version"),
+                stripe_account=options.get("stripe_account"),
+            )
+
+        params_with_filters = dict(self._retrieve_params)
+        params_with_filters.update({"page": self.next_page})
+        params_with_filters.update(params)
+
+        return await self._search_async(
+            **params_with_filters,
+        )
```

Comment:
Same here, move into a `_get_filters_for_next_page` method

Reply:
Done in [_get_url_for_search and _maybe_empty_result](https://github.com/stripe/stripe-python/pull/1219/commits/d38887b367d0b0cd8c63aef60debd6c86840e4d1)